lametro/management/commands/generate_tag_analytics.py

```diff
@@ -0,0 +1,36 @@
+import csv
+from tqdm import tqdm
+from django.core.management.base import BaseCommand
+
+from lametro.models import LAMetroBill
+
+
+class Command(BaseCommand):
+
+    help = "This command produces a CSV file that lists each Board Report's tags."
+
+    def handle(self, *args, **options):
+
+        with open('tag_analytics.csv', 'w', newline='') as f:
+            writer = csv.writer(f, delimiter=',')
+            files = []
```

Comment:
This works but it isn't necessarily optimal, because it will build up this list in memory. Can you think of a way to write the file in this first loop, instead of building up the list and writing it in a second loop?

Reply:
Very true-- this is a very inefficient way to do this. Thanks for pointing this out.